cdd headers ar in /usr/include in ArchLinux, /usr/include/cdd in Ubuntu and /usr/include/cddlib in Fedora

--- cmake2/FindCDD.cmake.orig	2019-01-29 15:13:37.000000000 +0100
+++ cmake2/FindCDD.cmake	2019-01-31 10:33:17.193453803 +0100
@@ -6,8 +6,8 @@
 #  CDD_DEFINITIONS - Compiler switches required for using CDD
 
 
-find_path(CDD_INCLUDE_DIR cdd/cdd.h
-          HINTS ${CDD_INCLUDEDIR} /usr/include
+find_path(CDD_INCLUDE_DIR cdd.h
+          HINTS ${CDD_INCLUDEDIR} /usr/include /usr/include/cdd /usr/include/cddlib
           PATH_SUFFIXES CDD )
 
 find_library(CDD_LIBRARY NAMES libcdd.so
--- src/CMakeLists.txt.orig	2019-01-05 01:53:00.000000000 +0100
+++ src/CMakeLists.txt	2019-01-31 22:19:07.673417759 +0100
@@ -19,6 +19,7 @@
 endif(USE_GLPK)
 
 ADD_LIBRARY(${LIBRARY_NAME} SHARED ${${LIBRARY_NAME}_SOURCES})
+TARGET_INCLUDE_DIRECTORIES(${LIBRARY_NAME} PUBLIC ${CDD_INCLUDE_DIRS})
 
 if (USE_GLPK)
   TARGET_LINK_LIBRARIES(${LIBRARY_NAME} ${GLPK_LIB})
--- CMakeLists.txt.orig	2019-01-05 01:53:00.000000000 +0100
+++ CMakeLists.txt	2019-01-31 22:25:41.837406946 +0100
@@ -41,6 +41,7 @@
 ADD_REQUIRED_DEPENDENCY(eigen3)
 ADD_REQUIRED_DEPENDENCY(hpp-centroidal-dynamics)
 ADD_REQUIRED_DEPENDENCY(hpp-spline)
+find_package(CDD REQUIRED)
 IF(USE_GLPK)
   find_package (glpk REQUIRED)
   add_definitions ( -DUSE_GLPK_SOLVER=1)
